app/views/examples/example_radios_checkboxes.html

```diff
@@ -64,15 +62,13 @@ <h1 class="heading-large">
           </fieldset>
         </div>
 
-        <h1 class="heading-large">
-          Where should we send your new<br> passport?
-        </h1>
-
         <div class="form-group">
           <fieldset>
 
-            <legend class="visually-hidden">
-              Where should we send your new passport?
+            <legend>
+              <h1 class="heading-large">
+                Where should we send your new passport?
```

Comment:
Ah, I just found it. It was there from the beginning of the checkbox examples in bc2c6b409c10f8f3ab210a1bb5d70ae721b991dd, so there is no further explanation as it was part of a bigger thing. Maybe @gemmaleigh knows more?